test/lib/cpd/CpdEngine.test.ts

```diff
@@ -52,22 +25,26 @@ describe('CpdEngine', () => {
 			const targets: RuleTarget[] = [{
 				target: '',
 				paths: ['file.cls','file.trigger','file.page','file.py','file.component']
-			}]
+			}];
 		
 			let langPathMap: Map<LANGUAGE, string[]> = new Map();
 			langPathMap.set(LANGUAGE.APEX, ['file.cls','file.trigger']);
 			langPathMap.set(LANGUAGE.VISUALFORCE, ['file.page','file.component']);
 			expect((testEngine as any).sortPaths(targets)).to.eql(langPathMap);
```

Comment:
@jfeingold35 @jbartolotta-sfdc As per your suggestions, I've restored the `as any` call for `sortPaths` and made it a private method again. The remaining `as any` calls are on public methods and they are already accessible directly.